make-options.json

```diff
@@ -291,6 +291,26 @@
     ],
     "Node20_229_2": [],
     "Node20_229_3": [],
-    "Node20_229_4": [],
+    "Node20_229_4": [
+        "AppCenterDistributeV1",
+        "AppCenterDistributeV2",
+        "AppCenterTestV1",
+        "AzureContainerAppsV0",
+        "AzureContainerAppsV1",
+        "AzureIoTEdgeV2",
+        "AzureStaticWebAppV0",
+        "AzureVmssDeploymentV0",
```

Comment:
This is pulling in a bunch of different teams -- my thought is to split into seperate groups.  E.g. Test (can be combined with RM), AppCenter, Other

Reply:
Yes this PR contains tasks that belongs various of teams, each team probably owns like 1~2 tasks, we have Node20_229_4 and Node20_229_5 for all rest of tasks. Node20_229_1 used by artifact packages, Node20_229_2 and Node20_229_3 are used by razvan's team.